microsoft-edge/webview2/concepts/developer-guide.md

```diff
@@ -0,0 +1,80 @@
+
+Ensure the Evergreen WebView2 Runtime is installed before using your WebView2 application.  For more information, navigate to [Deploying the Evergreen WebView2 Runtime][Webview2ConceptsDistributionDeployingEvergreenWebview2Runtime].  
+
+## Run regular compatibility tests
+
+When using the Evergreen WebView2 Runtime, ensure you run regular compatibility tests. Because the runtime is automatically updated, you'll want to ensure that your WebView2 application performs as expected.  For more information, navigate to [Stay compatible in Evergreen mode][Webview2ConceptsDistributionStayCompatibleEvergreenMode].  
+
+## Ensure APIs are supported by the installed WebView2 Runtime
+
+WebView2 applications need both a Webview2 SDK, and a WebView2 Runtime installed on the computer to run. Both the SDK and the runtime are versioned. Since APIs are continually being added to WebView2, new versions of the runtime are also released to support the new APIs. You'll need to ensure that the APIs used by your WebView2 application are supported by the WebView2 Runtime that's installed on the computer. 
+
+If you use the Evergreen WebView2 Runtime, consider the connectivity scenarios of your users. Some users may not be continually connected to the internet and may not use an updated runtime. When you push an update to your WebView2 application, the application may break because it uses newer APIs that are not available in the installed runtime. 
+ 
+To solve this situation, you can test for the availability of the APIs in the installed runtime, before making a call to the API. To test for API availability in the installed runtime, use:
+* The `queryinterface` in C/C++. For more information, navigate to <!--LINK-->
+* A try/catch block in .Net or WinUI, as displayed in the following code sample. 
+
+```csharp
+    /// Add code snippet that demonstrates how to use the try/catch block
+```
+    
+For more information, navigate to [Determine WebView2 Runtime requirement][Webview2ConceptsVersioningDetermineWebview2RuntimeRequirement].  
+
+## Update the Fixed Version Runtime  
+
+Depending on your environment, work stoppage due to applications going offline may significantly impact your business.  For example, certain financial or medical applications may have high availability requirements. In these scenarios, you may be required to use the Fixed Version Runtime to control updates to your application. If you use the Fixed Version Runtime, ensure you update your runtime regularly to reduce any potential security risk. When using 3rd-party content in Webview2 applications, always consider the content untrusted.  For more information, navigate to [Fixed Version distribution mode][Webview2ConceptsDistributionFixedVersionDistributionMode].  
+
+## Manage new versions  
```

Comment:
I think this needs to be more descriptive.
"Manage new versions of the runtime" maybe?

I wonder if this should be a subsection to "Use Evergreen" above.